### ISSUE TITLE
Broken in VSCode version 1.7.0

### ISSUE THREAD

**Issue by cryzed** [lostintangent/gitdoc#35]
This extension seemingly stopped working for me after updating to VSCode version 1.7.0. When I toggle the developer tools, I can see the following gitdoc-related error pop up after the configured commit delay:

```
mainThreadExtensionService.ts:80 TypeError: Cannot read properties of undefined (reading 'add')
	at vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:69:46
	at Generator.next (<anonymous>)
	at vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:8:71
	at new Promise (<anonymous>)
	at __awaiter (vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:4:12)
	at commit (vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:42:12)
	at vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:93:13
	at Generator.next (<anonymous>)
	at vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:8:71
	at new Promise (<anonymous>)
	at __awaiter (vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:4:12)
	at vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/out/watcher.js:92:61
	at Timeout.later [as _onTimeout] (vscode-file://vscode-app/home/user/.vscode-oss/extensions/vsls-contrib.gitdoc-0.0.8/node_modules/debounce/index.js:27:23)
	at listOnTimeout (node:internal/timers:557:17)
	at processTimers (node:internal/timers:500:7)
```

It looks like the [`_repository`-property](https://github.com/lostintangent/gitdoc/blob/8e11a8dfb8bab01477f356054d6bce8bf3d69dbb/src/watcher.ts#L76) isn't part of the API anymore. I think the change to the related APIs happened in [this](https://github.com/microsoft/vscode/commit/c7c0acd2ce4e0e541554fe8958922170a0fc53dc) commit.

**Comment by lostintangent** [comment]
Good catch! Yeah it looks like they just removed the leading underscore from the variable name. So hopefully this is a super simple fix 🤞

**Comment by cryzed** [comment]
By the way, the repository link on your Visual Studio Marketplace page is broken, it's missing a `/`.

**Comment by moalamri** [comment]
I removed the leading underscore and it worked again,
I had to rename the remote repository because I don't have the name as origin, perhaps you could add a setting for that?

**Comment by TarekVisch** [comment]
The extension still hasn't been updated to vs code marketplace, thus still not working with vs code 1.70

**Comment by lostintangent** [comment]
v0.1.0 has now been published, and fixed VS Code 1.70. Thanks for the patience here!
